server.js

```diff
@@ -1,38 +1,117 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import mongoose from 'mongoose';
+
+import booksData from './data/books.json';
+
+// Code for setting up our Mongo database
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+// Create Book model to start populating our database
+const Book = mongoose.model('Book', {
+  bookID: Number,
+  title: String,
+  authors: String,
+  average_rating: Number,
+  isbn: String,
+  isbn13: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+});
+
+// Function to start seeding our Database, will only run if
+// RESET_DB environment variable is present and is true
+if (process.env.RESET_DB) {
+  const seedDatabase = async () => {
+    // Starts by deleting any pre-existing Book objects to prevent duplicates
+    await Book.deleteMany({});
+    
+    // Creates a new Book instance for each book in the booksData
+    booksData.forEach((bookData) => {
+      new Book(bookData).save();
+    })
+  };
+  seedDatabase();
+};
+
+// Defines the port the app will run on
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
-// Start defining your routes here
+// ROUTES
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send('Hello world, welcome to Vane Bookish API, now powered by MongoDB!');
+});
+
+// Route to get all the books in the database
+app.get('/books', async (req, res) => {
+  const allBooks = await Book.find();
+  res.json(allBooks);
+});
+
+// Route to get a single book based on its ID
+app.get('/books/book/:bookID', async (req, res) => {
```

Comment:
Curious about the naming of the endpoint. I have one that does the same but I named it '/books/:id'. Did you name yours '/books/book/:id' so the user would know that the endpoint would return a specific book with that id? 😊 Nice to understand how you were thinking when you wrote the endpoint.